charmcraft/commands/store/__init__.py

```diff
@@ -999,10 +999,10 @@ def run(self, parsed_args: "Namespace") -> None:
                 emit.debug(f"Error when running PRIME step: {error}")
                 raise
 
-            from charmcraft.metafiles.actions import create_actions
+            from charmcraft.metafiles.metadata import create_metadata_yaml
```

Comment:
Oh my, I don't remember why I did these imports here :flushed: 

I sure hope they were because importing them at the module level was bad